Update README and package metadata for project housekeeping

thompsondt opened the pull request:
This pull request includes several housekeeping updates to improve project documentation and metadata:

README File Updates:
- Added project acknowledgements.
- Included detailed installation procedures for contributors.

package.json Metadata Updates:
- Updated the project license information.
- Added other relevant project details.

These changes aim to enhance the clarity and completeness of our project documentation and metadata, making it easier for contributors to get started and understand the project's background.

thompsondt committed:
chore: update package metadata

- the license should appear as MIT
- package information should include description, entrypoint, and keywords

thompsondt committed:
docs: update readme

- acknowledgements should be in their own section
- readme should contain a basic installation procedure for contributors

thompsondt committed:
Merge branch 'main' into housekeeping

thompsondt commented:
@CurtisGreen My bad. I should probably open this PR on dev. Please let me know if you have a preference.

CurtisGreen commented:
> @CurtisGreen My bad. I should probably open this PR on dev. Please let me know if you have a preference.

No that’s fine, it’s just saying I need to approve it. I guess I didn’t realize I put it under my other project.

I’m out today but will review when I’m home

CurtisGreen commented on README.md:
````diff
@@ -2,11 +2,30 @@
+    ```bash
+    npm install
+    ```
+3. Configure environment variables
+    ```bash
+    export NEXT_PUBLIC_MAPBOX_TOKEN=TOKEN_HERE
+    ```
````
Should put something about using this or the `.env` file

CurtisGreen replied:
And maybe link to mapbox site

thompsondt replied:
I can do this. I was fiddling with using .env.local last evening, trying to avoid storing secrets in plain text using 1Password. I had an issue with getting the secrets to de-reference. I am aware that `.env.local` will be ignored in commits and the plain text method does work. I will write something to this effect.

CurtisGreen replied:
I've always just used `.env`, so I'm not familiar with `.env.local` but whatever works

thompsondt replied:
These instructions are now updated to communicate multiple available methods for parameterizing environment variables.

thompsondt committed:
docs: update installation instructions

- contributors should know there are multiple methods availble for configuring environment variables

CurtisGreen commented:
@thompsondt It looks like I need to do some configuration changes to let it pass the authentication during a deploy. Hopefully won't be too bad
